lib/text.rb

```diff
@@ -0,0 +1,13 @@
+require 'twilio-ruby'
+
+def send_text(message)
+
+  @client = Twilio::REST::Client.new 
+  
+  @client.messages.create(
+    from: "+441273917629",
```

Comment:
Could think about removing personal phone number when committing to GH for privacy reasons.